app/models.py

```diff
@@ -2761,7 +2761,7 @@ def brief_id(self, brief_id):
     @validates('brief')
     def validates_brief(self, key, brief):
         if brief.status != "live":
-            raise ValidationError("Brief status must be 'live', not '{}'".format(brief.status))
+            raise ValidationError("Opportunity status must be 'live', not '{}'".format(brief.status))
```

Comment:
There is still some instances of brief in models.py. Check under `validates_questions_closed_at` , `validates_lot`, `validates_users `, `status `and `validates_questions_closed_at` function under the models.py